pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1995,3 +1995,56 @@ func makeQueuedPodInfos(num int, timestamp time.Time) []*framework.QueuedPodInfo
 	}
 	return pInfos
 }
+
+func TestPriorityQueue_calculateBackoffDuration(t *testing.T) {
+	type args struct {
+		podInfo *framework.QueuedPodInfo
+	}
+	tests := []struct {
+		name  string
+		queue *PriorityQueue
+		args  args
```

Comment:
Directly use podInfo as the field:

```suggestion
		podInfo *framework.QueuedPodInfo
```